tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -63,7 +63,7 @@ def montecarlo_radial1d(
     packet_seeds = montecarlo_configuration.packet_seeds
 
     number_of_vpackets = montecarlo_configuration.number_of_vpackets
-
+    if iteration == 0: montecarlo_main_loop.recompile() # Make sure we update
```

Comment:
this is dangerous and only needed for testing. Is that something we should have a config option on?

Reply:
I don't think this is dangerous, I think this is necessary for tardis to work properly if config options change between runs in the same kernel

Reply:
This is really just an initialization step